### PR TITLE
Add another orphan instance for Compose

### PR DESCRIPTION
**Pull request by madeline-os**
This instance is more in the same vein as the other Compose orphans in the same place, to make Vessels usable with obelisk/rhyolite.

I've also added a cabal.project file so that it's easier to work with Vessel as a dep inside of a larger project.

@Ericson2314 told me a bunch of stuff about how this is fucked up in base upstream. I have to go through his notes and  turn them into a plan that deals with this situation properly. :)

### CODE REVIEW

**Review by Ericson2314 — COMMENTED** [review]
~~This should go in `patch` not here, then it won't be an orphan~~ Oh I see it is with the others in that module, sorry.

**Review by ali-abrar — CHANGES_REQUESTED** [review]
Please update the changelog. If there's further work to be done (later, when base is updated, perhaps), maybe open an issue so we don't forget.

**Review by Ericson2314 — CHANGES_REQUESTED** [review]
We can put these in `patch` after since base-orphans has these after all https://hackage.haskell.org/package/base-orphans-0.8.6/docs/Data-Orphans.html#v:-36-fMonoidCompose

File: ChangeLog.md [hunk 1]
```diff
@@ -3,3 +3,7 @@
 ## 0.1.0.0
 
 * Initial release
+
+## 0.1.1.0
```
Comment (Ericson2314) [inline ChangeLog.md:7]: We typically have gone newest first, FYI.

**Pull request merged**